python/sglang/srt/mem_cache/memory_pool.py

```diff
@@ -415,6 +415,7 @@ def alloc(
                 req.mamba_pool_idx = mid
             if mid is not None:
                 mamba_index.append(mid)
+                req.mamba_pool_idx_reserved = None
```

Comment:
Do you think putting in `if req.mamba_pool_idx is not None:  # for radix cache` is cleaner?